package.json

```diff
@@ -1,27 +1,43 @@
 {
   "name": "@nebulr-group/nblocks-ts-client",
-  "version": "5.4.1",
-  "description": "This library contains the low level NBlocks client that is shipped with all of our NBlocks plug-n-play experiences for different technologies.",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "files": [
+  "version": "5.4.2",
```

Comment:
These are breaking changes. Version should be `6.0.0`, start with `6.0.0-beta.1` so we can get this out to npm as a beta package first

Reply:
WIll do!